UI/data/locale/en-US.ini

```diff
@@ -402,6 +402,7 @@ Output.ConnectFail.Title="Failed to connect"
 Output.ConnectFail.BadPath="Invalid Path or Connection URL. Please check your settings to confirm that they are valid."
 Output.ConnectFail.ConnectFailed="Failed to connect to server"
 Output.ConnectFail.InvalidStream="Could not access the specified channel or stream key, please double-check your stream key. If it is correct, there may be a problem connecting to the server."
+Output.ConnectFail.HdrDisabled="HDR output over RTMP is currently disabled for this codec"
```

Comment:
The message should not specify RTMP since any output (protocol) could use `OBS_OUTPUT_HDR_DISABLED`.

Reply:
Updated.